packages/in_app_purchase/lib/src/in_app_purchase/in_app_purchase_connection.dart

```diff
@@ -80,64 +108,86 @@ abstract class InAppPurchaseConnection {
-  /// The `autoConsume` if `true` by default, and we will call [consumePurchase] after a successful purchase for you. If you'd like to have an advance purchase flow management. You should set it to `false` and
-  /// consume the purchase when you see fit. Fail to consume a purchase will cause user never be able to buy the same item again. Setting this to `false` on iOS will throw an `Exception`.
-  ///
-  /// This method does not return anything. Instead, after triggering this method, purchase updates will be sent to [purchaseUpdatedStream].
-  /// You should [Stream.listen] to [purchaseUpdatedStream] to get [PurchaseDetails] objects in different [PurchaseDetails.status] and
-  /// update your UI accordingly. When the [PurchaseDetails.status] is [PurchaseStatus.purchased] or [PurchaseStatus.error], you should deliver the content or handle the error, then call
-  /// [completePurchase] to finish the purchasing process.
-  ///
-  /// You can find more details on testing payments on iOS [here](https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/StoreKitGuide/Chapters/ShowUI.html#//apple_ref/doc/uid/TP40008267-CH3-SW11).
-  /// You can find more details on testing payments on Android [here](https://developer.android.com/google/play/billing/billing_testing).
+  /// Consumable items can be "consumed" to mark that they've been used and then
+  /// bought additional times. For example, a health potion.
+  ///
+  /// To restore non consumable purchases across devices, you should keep track
+  /// of those purchase on your own server and restore the purchase for your
+  /// users. Consumed products are no longer considered to be "owned" by payment
+  /// platforms and will not be delivered by calling [queryPastPurchases].
+  ///
+  /// Consumable items are defined differently by the different underlying
+  /// payment platforms, and there's no way to query for whether or not the
+  /// [ProductDetail] is a consumable or not at runtime. On iOS, products are
+  /// defined as non consumable items in the [App Store
+  /// Connect](https://appstoreconnect.apple.com/login). [Google Play
+  /// Console](https://play.google.com/) products are considered consumable if
+  /// and when they are actively consumed manually.
+  ///
+  /// `autoConsume` is provided as a utility for Android only. It's meaningless
+  /// on iOS because the App Store automatically considers all purchases
+  /// "consumed" once the initial transaction is complete. `autoConsume` is
+  /// `true` by default, and we will call [consumePurchase] after a successful
+  /// purchase for you so that Google Play considers a purchase consumed after
+  /// the initial transaction, like iOS. If you'd like to manually consume
+  /// purchases in Play, you should set it to `false` and manually call
+  /// [consumePurchase] instead. Failing to consume a purchase will cause user
+  /// never be able to buy the same item again. Failing to manually Setting this
```

Comment:
"Failing to manually Setting this to `false` on iOS will throw an `Exception`."

Should it be "Setting this to `false` on iOS will throw an `Exception`. " ?